packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/service/ServiceEditorState.ts

```diff
@@ -153,6 +180,43 @@ export class ServiceEditorState extends ElementEditorState {
     this.selectedTab = SERVICE_TAB.TEST;
   }
 
+  get selectedOwnership(): ServiceOwnerOption | undefined {
+    const ownership = this.service.ownership;
+    if (ownership instanceof DeploymentOwnership) {
+      return {
+        label: 'Deployment',
```

Comment:
lets move this string as const on top as we are repeating them. 

so like const DEPLOYMENT = 'deployment'

Reply:
done